configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.yml

```diff
@@ -0,0 +1,40 @@
+Collections:
+- Name: hrnet_animalpose
+  Metadata:
+    Architecture:
+    - HRNet
+  README: configs/animal/2d_kpt_sview_rgb_img/topdown_heatmap/animalpose/hrnet_animalpose.md
+  Paper:
+  - Deep high-resolution representation learning for human pose estimation
+  - Cross-Domain Adaptation for Animal Pose Estimation
+Models:
+- Name: animal--2d_kpt_sview_rgb_img--topdown_heatmap--animalpose--hrnet_w32_animalpose_256x256
```

Comment:
The delimiter in Name is '--', please double check if it is intended.

Reply:
This is intended to replace '/'.